holidata/holidays/hr-HR.py

```diff
@@ -7,18 +7,19 @@
 class hr_HR(Locale):
     """
     01-01: [NF] Nova Godina
-    01-06: [NRF] Bogojavljanje / Sveta tri kralja
+    01-06: [NRF] Bogojavljanje ili Sveta tri kralja
     05-01: [NF] Praznik rada
     06-22: [NF] Dan antifašističke borbe
     06-25: [NF] Dan državnosti
-    08-05: [NF] Dan domovinske zahvalnosti
+    08-05: [NF] Dan pobjede i domovinske zahvalnosti i Dan hrvatskih branitelja
     08-15: [NRF] Velika Gospa
     10-08: [NF] Dan neovisnosti
-    11-01: [NRF] Dan svih svetih
+    11-01: [NRF] Svi sveti
+    11-18: [NRF] Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje
     12-25: [NRF] Božić
     12-26: [NRF] Sveti Stjepan
     Easter: [NRV] Uskrs
-    1 day after Easter: [NRV] Uskršnji ponedjeljak
+    1 day after Easter: [NRV] Uskrsni ponedjeljak
```

Comment:
Typo fix? At least wikipedia asks " Did you mean 'Uskrsni ponedjeljak'" (Mislili ste: ...) when searching for "Uskršnji ponedjeljak"

Reply:
> Typo fix? At least wikipedia asks " Did you mean 'Uskrsni ponedjeljak'" (Mislili ste: ...) when searching for "Uskršnji ponedjeljak"

Well, according to the official [law](https://narodne-novine.nn.hr/clanci/sluzbeni/2019_11_110_2212.html) document, the correct version is "Uskrsni ponedjeljak".